build/yamls/antrea-octant.yml

```diff
@@ -0,0 +1,62 @@
+# yaml template for octant and antrea-octant-plugin
+# Right config parameters and Docker image must be specified.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: antrea-octant
+  namespace: kube-system
+  labels:
+    app: antrea
+    component: antrea-octant
+spec:
+  replicas: 1
+  strategy:
+    # Ensure the existing Pod is killed before the new one is created.
+    type: Recreate
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-octant
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-octant
+    spec:
+      priorityClassName: system-cluster-critical
+      tolerations:
+        # Mark it as a critical add-on.
+        - key: CriticalAddonsOnly
+          operator: Exists
+        # Allow it to schedule onto master nodes.
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      containers:
+        - name: antrea-octant
+          image: octant-antrea-ubuntu
+          imagePullPolicy: IfNotPresent
+          # Octant start-up env
+          env:
+            # 0.0.0.0 allows all hosts
+            - name: OCTANT_ACCEPTED_HOSTS
+              value: "0.0.0.0"
+            # Port 80 can be changed according to your set up.
+            - name: OCTANT_LISTENER_ADDR
+              value: "0.0.0.0:80"
+            - name: OCTANT_DISABLE_OPEN_BROWSER
+              value: "true"
+            # Change admin.conf to the name of kubeconfig file in your set up.
+            - name: KUBECONFIG
+              value: "/kube/admin.conf"
+          command: ["octant"]
+          args: ["-v"]
+          ports:
+          - containerPort: 80
```

Comment:
I suggest having `hostPort: 80` as a placeholder here. This is what people are likely to change (they probably don't care as much about `OCTANT_LISTENER_ADDR`, since Octant is running inside it's own container).

Reply:
`hostPort` is not a recommended way to expose a service via host ip, see reason here: https://kubernetes.io/docs/concepts/configuration/overview/#services. It would also be inconvenient for user to know which IP should be used to access it.
A `NodePort` type service might be suitable for such case.

Reply:
@antoninbas @tnqn , thanks a lot for the comments. I have done some further investigation for this part. It seems that field hostPort does not work if it is a deployment. The field can only work if it is a Pod. And containerPort here is just a flag for noticing. So if we do not use hostNetwork, we have to use other way to expose the service such as NodePort service as suggested. I have updated the patch by adding the service. Please help take a look.